<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tree</string>
  </property>
  <widget class="QWidget" name="mainContainer">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1000">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="horizontalSpacing">
          <number>-1</number>
         </property>
         <item row="7" column="0">
          <widget class="QSlider" name="timeSetter">
           <property name="toolTip">
            <string>Current date</string>
           </property>
           <property name="statusTip">
            <string>Current date</string>
           </property>
           <property name="whatsThis">
            <string>Current date</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>8760</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <widget class="QSlider" name="timeStep">
           <property name="toolTip">
            <string>Current year</string>
           </property>
           <property name="statusTip">
            <string>Current year</string>
           </property>
           <property name="whatsThis">
            <string>Current year</string>
           </property>
           <property name="maximum">
            <number>8760</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QSlider" name="auxin">
           <property name="toolTip">
            <string>auixin level</string>
           </property>
           <property name="statusTip">
            <string>auixin level</string>
           </property>
           <property name="whatsThis">
            <string>auixin level</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="auxinLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="deltaTemperatureLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="timeStepVal">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="angles">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QSlider" name="temperatureSetter">
           <property name="statusTip">
            <string>Current temperature</string>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="temperature">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QSlider" name="avgTemperature">
           <property name="toolTip">
            <string>avg. temperature</string>
           </property>
           <property name="statusTip">
            <string>avg. temperature</string>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
           <property name="sliderPosition">
            <number>150</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="avgTemperatureVal">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QSlider" name="year">
           <property name="toolTip">
            <string>Time Step (in hours)</string>
           </property>
           <property name="statusTip">
            <string>Time Step (in hours)</string>
           </property>
           <property name="whatsThis">
            <string>Time Step (in hours)</string>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QSlider" name="water">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="time">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="positionLabel">
           <property name="text">
            <string>Coords</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="yearLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QSlider" name="deltaTemperature">
           <property name="statusTip">
            <string>max difference between temperatures</string>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="sliderPosition">
            <number>600</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="anglesLabel">
           <property name="text">
            <string>Angles:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="0">
          <widget class="QSlider" name="horizontalSlider_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="positions">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QSlider" name="horizontalSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QSlider" name="segmentAuxin">
           <property name="toolTip">
            <string>segmentAuxin</string>
           </property>
           <property name="statusTip">
            <string>segmentAuxin</string>
           </property>
           <property name="whatsThis">
            <string>segmentAuxin</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLabel" name="auxinValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLabel" name="segmentAuxinLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QSlider" name="segmentAuxinProd">
           <property name="toolTip">
            <string>segmentAuxinProd</string>
           </property>
           <property name="statusTip">
            <string>segmentAuxinProd</string>
           </property>
           <property name="whatsThis">
            <string>segmentAuxinProd</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="22" column="0">
          <widget class="QSlider" name="horizontalSlider_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QLabel" name="budAuxinLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QSlider" name="budAuxinProd">
           <property name="toolTip">
            <string>budAuxinProd</string>
           </property>
           <property name="statusTip">
            <string>budAuxinProd</string>
           </property>
           <property name="whatsThis">
            <string>budAuxinProd</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QSlider" name="budAuxin">
           <property name="toolTip">
            <string>budAuxin</string>
           </property>
           <property name="statusTip">
            <string>budAuxin</string>
           </property>
           <property name="whatsThis">
            <string>budAuxin</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QSlider" name="leafAuxin">
           <property name="toolTip">
            <string>leafAuxin</string>
           </property>
           <property name="statusTip">
            <string>leafAuxin</string>
           </property>
           <property name="whatsThis">
            <string>leafAuxin</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QLabel" name="budAuxinProdLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QLabel" name="segmentAuxinProdLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="22" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QLabel" name="leafAuxinLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QPushButton" name="resetTree">
           <property name="text">
            <string>Reset tree</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Glade" name="glade" native="true">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Glade</class>
   <extends>QWidget</extends>
   <header>glade.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>temperatureSetter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>temperature</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>avgTemperature</sender>
   <signal>valueChanged(int)</signal>
   <receiver>avgTemperatureVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deltaTemperature</sender>
   <signal>valueChanged(int)</signal>
   <receiver>deltaTemperatureLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
